docs/api/alerting/create_rule.asciidoc

```diff
@@ -1,19 +1,29 @@
 [[create-rule-api]]
-=== Create rule API
+== Create rule API
 ++++
 <titleabbrev>Create rule</titleabbrev>
 ++++
 
 Create {kib} rules.
 
 [[create-rule-api-request]]
-==== Request
+=== {api-request-title}
 
 `POST <kibana host>:<port>/api/alerting/rule/<id>`
 
 `POST <kibana host>:<port>/s/<space_id>/api/alerting/rule/<id>`
 
-==== {api-description-title}
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Management* > *Stack Rules* feature or
+for the *Discover*, *{ml-app}*, *{observability}*, or *Security* features,
```

Comment:
> for the *Discover*...

From my tests, it looks like `all` privileges for the Discover feature aren't sufficient to create rules in the Discover app. Is that a bug or should I omit that from this list?